tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -151,6 +151,38 @@ public function test_are_site_fields_synced() {
 		$this->assertEquals( $failure_response_data, false );
 	}
 
+	public function test_url_handles_staging() {
+		$url = $this->google_proxy->url();
+		$this->assertEquals( $url, Google_Proxy::PRODUCTION_BASE_URL );
+		// The test for this behaviour depends on a constant value which can
+		// only be redefined if PECL extension runkit7 is installed.
+		if ( defined( 'GOOGLESITEKIT_PROXY_URL' ) ) {
+			if ( function_exists( 'runkit7_constant_remove' ) ) {
+				runkit7_constant_remove( 'GOOGLESITEKIT_PROXY_URL' );
+			} else {
+				return;
+			}
+		}
+		define( 'GOOGLESITEKIT_PROXY_URL', Google_Proxy::STAGING_BASE_URL );
+		$url = $this->google_proxy->url();
+		$this->assertEquals( $url, Google_Proxy::STAGING_BASE_URL );
+	}
+
+	public function test_url_ignores_invalid_values() {
+		// The test for this behaviour depends on a constant value which can
+		// only be redefined if PECL extension runkit7 is installed.
+		if ( defined( 'GOOGLESITEKIT_PROXY_URL' ) ) {
+			if ( function_exists( 'runkit7_constant_remove' ) ) {
+				runkit7_constant_remove( 'GOOGLESITEKIT_PROXY_URL' );
+			} else {
+				return;
+			}
+		}
```

Comment:
Same here. Let's mark this test as skipped if the `runkit7` extension isn't loaded.

Reply:
https://github.com/google/site-kit-wp/pull/3495/commits/bfad37b1860e628aded628a113f35601031ba929